src/main/java/me/eccentric_nz/TARDIS/commands/tardis/TARDISVersionCommand.java

```diff
@@ -33,13 +38,27 @@
     }
 
     boolean displayVersion(CommandSender sender) {
-        String version = plugin.getPM().getPlugin("TARDIS").getDescription().getVersion();
-        String chunkversion = plugin.getPM().getPlugin("TARDISChunkGenerator").getDescription().getVersion();
+        final PluginManager pm = plugin.getPM();
```

Comment:
Not sure why you need this at all (and my original code is not much better)...
`String tardisversion = plugin.getDescription().getVersion();`
and
`String chunkversion = plugin.getPM().getPlugin("TARDISChunkGenerator").getDescription().getVersion();`